spec/lob/resources/us_zip_lookups_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "spec_helper"
+
+describe Lob::Resources::USZipLookups do
+
+  before :each do
+    @sample_params = {
+      zip_code: "94107"
+    }
+  end
+
+  subject { Lob::Client.new(api_key: API_KEY) }
+
+  describe "lookup" do
+    it "should lookup a US zip code" do
+      result = subject.us_zip_lookups.lookup @sample_params
```

Comment:
With the above change, we would just need to change this to `subject.us_zip_lookups.create @sample_params`